forch/__main__.py

```diff
@@ -0,0 +1,106 @@
+"""Entry point for orchestrator component for controlling a Faucet SDN"""
+
+import argparse
+import functools
+import logging
+import os
+import sys
+import yaml
+
+import forch.faucet_event_client
+from forch.forchestrator import Forchestrator
+import forch.http_server
+from forch.utils import configure_logging
+
+from forch.__version__ import __version__
+
+LOGGER = logging.getLogger('forch')
+
+_FORCH_CONFIG_DEFAULT = 'forch.yaml'
+
+
+def load_config():
+    """Load configuration from the configuration file"""
+    # TODO: 1) use protobuf after entire forch config is converted
```

Comment:
each TODO should be it's own single line

Reply:
Done